Network/dedicated-load-balancer-firmware-upgrade.md

```diff
@@ -0,0 +1,231 @@
+{{{
+  "title": "Dedicated Load Balancer Firmware update",
+  "date": "12-11-2015",
+  "author": "Maxim Volkov",
+  "attachments": [],
+  "related-products" : [],
+  "contentIsHTML": false,
+  "sticky": false
+}}}
+
+### Introduction
+
```

Comment:
overall comment, should there be a section for troubleshooting (at a minimum how they get help if something goes wrong) and how to confirm the upgrade is complete?  

lastly, have you validated the NOC wants customers upgrading their own firmware as the default option?  my experience getting multiple upgrades is they did it on request (I didn't have to).  i'd think a customer would prefer that model so if its an option we should present the customer the details on how to get the upgrade done by our teams.  this kb is still good to show but we should make sure we at least let them know there is another choice.